packages/shader-shaderlab/src/shaders/PBR.gs

```diff
@@ -58,6 +58,10 @@ Shader "PBR.gs" {
       Header("Transmission"){
         material_Transmission("Transmission", Range(0, 1, 0.01)) = 0;
         material_TransmissionTexture("TransmissionTexture", Texture2D);
+        material_AttenuationColor("AttenuationColor", Color ) = (1, 1, 1, 1);
+        material_AttenuationDistance("AttenuationDistance", Range(0, 1, 0.01)) = 0;
+        material_Thickness("Thickness", Range(0, 5, 0.01)) = 0;
+        material_ThicknessTexture("ThicknessTexture", Texture2D);
```

Comment:
The default values ​​of `material_AttenuationDistance` and `material_Thickness` also seem to be arbitrary. Please carefully consider their reasonable default values.